src/MainWindow.cpp

```diff
@@ -2393,6 +2419,20 @@ void MainWindow::showDataColumnPopupMenu(const QPoint& pos)
     popupBrowseDataHeaderMenu->exec(ui->dataTable->horizontalHeader()->mapToGlobal(pos));
 }
 
+void MainWindow::showRecordPopupMenu(const QPoint& pos)
+{
+    const QString sCurrentTable = ui->comboBrowseTable->currentText();
```

Comment:
Second thing is this. I used this hint (which already took place in the code before me) to get current table. Getting something through UI widgets - really hurts. I think you guys should make some workaround for things like this in tablemodel class.
If you want, I can send another patch for this stuff later.